app/views/user_mailer/recipient_notification.text.erb

```diff
@@ -3,7 +3,7 @@
 
 A gift work has been posted for you<% if @collection %> in the "<%= @collection.title %>" collection (<%= collection_url(@collection) %>)<% end %> at the Archive of Our Own!
 
-"<%= @work.title.html_safe %>" (<%= @work.word_count %> words)
+"<%= @work.title.html_safe %>"‎ (<%= @work.word_count %> words)
```

Comment:
This seems to be using the RTL mark (U+200F) while the other emails use the LTR mark (U+200E).